plugin/pkg/shoot/dns/admission_test.go

```diff
@@ -119,6 +119,19 @@ var _ = Describe("dns", func() {
 			shootCopy.Spec.SeedName = nil
 			shootBefore := shootCopy.DeepCopy()
 
+			attrs := admission.NewAttributesRecord(shootCopy, nil, core.Kind("Shoot").WithVersion("version"), shootCopy.Namespace, shootCopy.Name, core.Resource("shoots").WithVersion("version"), "status", admission.Create, &metav1.CreateOptions{}, false, nil)
+
+			err := admissionHandler.Admit(context.TODO(), attrs, nil)
+
+			Expect(err).NotTo(HaveOccurred())
+			Expect(*shootCopy).To(Equal(*shootBefore))
+		})
+
+		It("should do nothing because the shoot status is updated", func() {
```

Comment:
I think the test case names are switched. This one needs to be:
```suggestion
		It("should do nothing because the shoot does not specify a seed (create)", func() {
```

And the above test case needs to be 
```
It("should do nothing because the shoot status is updated", func() {
```

Reply:
Yes, makes sense. Thank you 👍